sway-lib-std/src/address.sw

```diff
@@ -106,8 +106,9 @@ impl Address {
         while index < number_of_outputs {
             if let Output::Variable = output_type(index) {
                 if output_amount(index) == 0 {
-                    asm(r1: self.value, r2: index, r3: amount, r4: asset_id.value) {
-                        tro r1 r2 r3 r4;
+                    asm(r1: self.value, r2: index, r3: amount, r4: asset_id
+                        .value) {
```

Comment:
This line break doesn't look right. I think it should either stay the same or look like this:

```
                    asm(
                        r1: self.value,
                        r2: index,
                        r3: amount,
                        r4: asset_id.value
                    ) {
```